.github/workflows/tests.yml

```diff
@@ -0,0 +1,59 @@
+# Generated from:
+# https://github.com/zopefoundation/meta/tree/master/config/pure-python
```

Comment:
Was this generated withe `zopfoundation/meta` or is this just copy/paste?

Reply:
Hm, just copy-paste.